docs/writing/posts/learning.md

```diff
@@ -0,0 +1,153 @@
+
+## Leverage
+
+My whole strategy for learning can be summarized into a set of tactics. I worked very hard in the beginning to get a bit ahead, built a peer group that is a few steps behind me, and then taught them. Rinse and repeat. Their role in this ecosystem is to bring me the edge cases I wouldn't have thought of, and my job is to learn alongside them, always a few steps ahead, and then teach them.
+
+Even if I'm only two weeks ahead, helping them out means spending a few hours helping them get two weeks ahead. If you're two weeks ahead and spend one hour teaching 30 people, you just provided 60 human weeks of value and years of value in one hour. That's a pretty good exchange rate.
+
+## The real world
+
+That basically summarizes how I thought about learning in a school setting where because roughly speaking, everyone is somewhere on the assembly line, the system I used to make progress in new hobbies was very differen since it's rare to enter a new place as an adult and find you have access to a large group of people who are all trying to learn the same thing with the same level of intensity as a group of high achieving students, many of those same people will not pursuit any hobbies as they decide to focus on their careers.
+
+The following section will mostly be some woo woo so I'm going to premptively apologize to the hacker news crowd.
+
+### Pottery
+
+The biggest thing I learned in pottery, except for the fact that I sucked for like two years, was how we measured progress. I just remember thinking that I wanted to go to class and leave with six nice cups. I would make a couple of cups, but they would break somewhere in the process, and that would feel really disheartening. I remember telling my teacher what was going on, and they gave me advice that stuck with me forever. They said, "You should not focus on how many cups you make; you should just count how many bags of clay you go through." This was a huge perspective shift for me because a cup was 250 g, while a bag of clay was 25 kg. If I had considered the bags of clay as the metric, my progress would have been a straight `0-0` for the last 2 months.
+
+#### Process vs Outcome
+
+I was holding the work a little too precious, and I wasn't actually putting in any volume.
+
+What did I learn? I basically learned that we often set outcome-based goals instead of process-based goals, and even when we do set process-based goals, we are definitely not using the right measurement. No wonder I wasn't doing enough to actually achieve the outcomes that I wanted. Moreover, I was also not measuring the right thing, usually the units and order of manitude were off and figuring out the right metric bags of clay was a huge part of the process.
```

Comment:
Moreover, I was also not measuring the right thing -- usually the units and order of magnitude were off and figuring out the right metric of bags of clay was a huge part of the process.